src/partitioned-dbs/index.rst

```diff
@@ -105,14 +105,20 @@ sensor readings from an IoT field monitoring service. In this particular
 use case, it's quite logical to group all documents by their ``sensor_id``
 field. In this case, we would call the ``sensor_id`` the partition.
 
+A partition offers two benefits. Firstly, it allows you to simplify the
+workload on the query engine, by allowing the engine to only query shards
```

Comment:
I'm not sure "simplify the workload" is clearer here. What it's doing is allowing the user to group data in a way that is understood _a priori_, a.k.a. ahead of time. Since most searches (theoretically) are being done only within one given partition, queries are _optimised_ by only having to run within any one given partition at a time.